web/webplatform/VERSION

```diff
@@ -0,0 +1 @@
+2.29.1
```

Comment:
Question: any plan to make this dynamic?

Reply:
nope. we need to be explicit.

Reply:
ok, then please add a document in the catcafe to clarify who does what, how, and when / how often.